resources/views/home.blade.php

```diff
@@ -0,0 +1,23 @@
+@extends('layouts.app')
+
+@section('content')
+<div class="container">
+    <div class="row justify-content-center">
+        <div class="col-md-8">
+            <div class="card">
+                <div class="card-header">{{ trans('shopping.home.dashboard') }}</div>
```

Comment:
những chỗ này nên dùng @lang đi em

Reply:
theo em thì @trans và @lang thì khác nhau như nào và dùng như nào cho thích hợp

Reply:
@trans: Nếu tham số truyền vào không có trong file dịch thì nó sẽ in ra tham số đó. @tran có thể dùng trong file php
@lang thì sẽ không in ra gì cả, chỉ dùng trong file .blade.
Em thường dùng trans hơn, vậy nên e cũng không rõ sự khác nhau giữa chúng ạ. 
Anh giải thích thêm giúp em ạ.

Reply:
dùng @lang sẽ ngắn gọn hơn do đó những chỗ chỉ cần hiển thị ra text thì nên dùng @lang. chỉ được sử dụng Blade templating engine tức file view .blade.php. tuy nhiên trong trường hợp lấy ra dữ liệu thì nên dùng @trans. 
Vậy @trans ta nên dùng cho form

Reply:
Vâng a. A merge giúp em với ạ.